mvp-spec/partials-updater/src/files.ts

```diff
@@ -0,0 +1,45 @@
+import RootPath from "app-root-path";
+import fs from "fs";
+import path from "path";
+
+const PartialsDir = path.join(RootPath.path, "..", "partials");
+const DocumentsDir = path.join(RootPath.path, "..");
+const DocumentExts = [ ".md" ]
+
+export const getPartialPath = (file: string): string => path.join(PartialsDir, file);
+
+/** Load a partial based on its name. */
+export const loadPartial = (file: string): string => loadFile(file, PartialsDir);
+
+/** Load a document based on its name. */
+export const loadDocument = (file: string): string => loadFile(file, DocumentsDir);
+
+/** List the documents by names. */
+export const listDocuments = (): string[] => listFiles(DocumentsDir, isDocument);
+
+/** Rewrite a document. */
+export function rewriteDocument(name: string, content: string) {
+    const fullPath = path.join(DocumentsDir, name);
+    if (!fs.existsSync(fullPath)) {
```

Comment:
NIT: since you're using async in other places of the code, you could use the async versions of `fs` in this file as well.

Reply:
Done